roles/get_bmc_facts/tasks/main.yml

```diff
@@ -16,5 +16,10 @@
 
 - name: Extract all FW versions to facts
   ansible.builtin.set_fact:
```

Comment:
this looks like a separate change we want to merge first, can you do separate PR for i t?